src/com/vivian/java/leetcode/Q1_TwoSum.java

```diff
@@ -14,5 +16,23 @@
  * Because nums[0] + nums[1] = 2 + 7 = 9, return [0, 1].
  */
 public class Q1_TwoSum {
+    public int[] twoSum(int[] nums, int target) {
+        if (nums == null) {
+            return null;
+        }
+        HashMap<Integer, Integer> hs = new HashMap<>();
```

Comment:
It's better to program to an interface, instead of to program to an implementation. 
i.e. you code can be replaced with
**Map**<Integer, Integer> hs = new HashMap<>();

Imagine next time you found out that maybe it's better to use TreeMap (which implements Map), all you have to change is 1 place (i.e. change new HashMap to new TreeMap).